src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/ManagedIndexCoordinator.kt

```diff
@@ -293,10 +289,11 @@ class ManagedIndexCoordinator(
         indexToMatchedPolicy.filterNotNullValues()
             .forEach { (index, policyID) ->
                 val indexUuid = indexMetadatas[index].indexUUID
-                if (indexUuid != null) {
-                    logger.info("index [$index] will be managed by policy [$policyID]")
+                val ismTemplate = templates[policyID]
+                if (indexUuid != null && ismTemplate != null) {
```

Comment:
Do you want a warning for the else clause here?

Reply:
add a warning log:
```
else {
    logger.warn("index [$index] with null index uuid [$indexUuid] or " +
        "matching template [$ismTemplate] for this index is null.")
}
```